grafana/agent
# Add Initial Windows Service Support

#### PR Description 

Add initial support for Windows Service. Now without the windows installer script this isn't terribly useful. But instead lays the framework. 

#### Which issue(s) this PR fixes 

Closes #444  Windows Native - Allow native running in service mode 

#### Notes to the Reviewer

Steps to test

NOTE the all important space after binpath=
```
sc create Agent binpath= "<path to agent>\agent.exe -config.file=<path to agent config>\agent-config.yaml"
sc start Agent
sc query Agent
```

Then check localhost:12345/-/healthy

#### PR Checklist
